src/index.js

```diff
@@ -113,6 +121,14 @@ class Urls {
   servicesManifest() {
     return `${this.rootUrl}/references/manifest.json`;
   }
+
+  /**
+   * Returns a link to the task group or the task in Task Inspector
+   */
+  taskInspector(taskGroupId, taskId) {
+    let taskLog = taskId ? `/tasks/${taskId}/runs/0/logs/public/logs/live.log` : '';
```

Comment:
I wondered myself and opened the app - but I don't have a valid taskGroup ID or task ID to really see. I haven't look into the code. But of course, since it's not just another deployment of an existing app but a completely separate app - if this particular URL part is not an API call defined somewhere in a backend codebase which both apps use, it can be anything in that other app.

I am very much tempted to close this PR. I don't think I know the URL part of Taskcluster architecture enough to make suggestions to this repo.

I'll make a PR with this commit tho https://github.com/taskcluster/taskcluster-lib-urls/pull/18/commits/a63f0901f1d1233f847ecbe151ea589b9c152247 because the javascript tests currently don't run.

Reply:
Well, what am I saying - I think in the web apps, the URLs that people see in their browser are defined on the front-end mostly.

I think I I understand now why there is a `ui` function in here and why it's so generic - it's just for rootUrl and nothing else. I'm still not clear on the destiny of the "legacy" root URLs (are they gone? are they still around here and there? I remember vaguely the problem was that these "root URLs" were just hard-coded in the codebase everywhere?), but I think I'll close this PR.